app/server/database/schema/readme.md

```diff
@@ -0,0 +1,43 @@
+# User Data Schema/Model
+
+The model/schema for the users has been designed inn the table below.  
+Please note: MongoDB automaticaly assigns an objectID to every document/user object entered so we can use that as the primary key for the document.
+
+| Field          | Data Type | Description                                        |
+| -------------- | --------- | -------------------------------------------------- |
+| userName       | String    | User Login Name, can serve as primary key          |
+| userPassword   | String    | User Password, may be encrypted for security       |
+| userFirstName  | String    | Users First Name                                   |
+| userLastName   | String    | Users Last Name                                    |
+| userMiddleName | String    | Users Middle Name, will be optional                |
+| userEmail      | String    | Users email address                                |
+| userGoals      | JS Array  | An array of strings to store all goals             |
+| userTasks      | JS Array  | An array of strings to store all tasks             |
+| userJournal    | URL       | Contain a link to the users journal implementation |
+| userMood       | JS Object | An object to help model the users moods            |
+
+# userMood Object Structure
```

Comment:
This also may be best to update later, but I think a user should have an array of moods, and each mood should be associated with a date on the calendar.

What do you think @sontushkpaul ?